hvpy/helpers.py

```diff
@@ -124,3 +126,83 @@ def createMovie(
         overwrite=overwrite,
     )
     return Path(filename)
+
+
+@_add_shared_docstring(takeScreenshotInputParameters)
+def createScreenshot(
+    date: datetime,
+    imageScale: float,
+    layers: str,
+    events: Optional[str] = None,
+    eventLabels: Optional[bool] = False,
+    scale: Optional[bool] = False,
+    scaleType: Optional[str] = None,
+    scaleX: Optional[int] = None,
+    scaleY: Optional[int] = None,
+    width: Optional[str] = None,
+    height: Optional[str] = None,
+    x0: Optional[str] = None,
+    y0: Optional[str] = None,
+    x1: Optional[str] = None,
+    y1: Optional[str] = None,
+    x2: Optional[str] = None,
+    y2: Optional[str] = None,
+    watermark: Optional[bool] = False,
+    overwrite: bool = False,
+    filename: Union[str, Path] = None,
+) -> Path:
+    """
+    Automatically creates a screenshot using `takeScreenshot` functions.
```

Comment:
```suggestion
    Automatically creates a screenshot using `takeScreenshot` function.
```

Reply:
Oh, I missed writing downloadScreenshot in docstring. But we can get binary data with display=True in the takeScreenshot, with this we dont have to use downloadScreenshot. Should we use only takeScreenshot or use downloadScreenshot?

Reply:
What's the difference?

Reply:
No difference.

Reply:
Use download